src/CommunityToolkit.Maui.Camera/CommunityToolkit.Maui.Camera.csproj

```diff
@@ -53,21 +53,6 @@
   <ItemGroup Condition="$(TargetFramework.Contains('-android'))">
     <PackageReference Include="Xamarin.AndroidX.Camera.View" Version="1.3.1.1" />
     <PackageReference Include="Xamarin.AndroidX.Camera.Camera2" Version="1.3.1.1" />
-
-    <!-- Xamarin.AndroidX.Camera 1.3.1.1 depends on Xamarin.AndroidX.Lifecycle.LiveData 2.7.0.1:
-    https://www.nuget.org/packages/Xamarin.AndroidX.Camera.Core/#dependencies-body-tab 
-    However, Dotnet MAUI still implicitly references Xamarin.AndroidX.Lifecycle.LiveData 2.6.1.3:
-    https://github.com/dotnet/maui/blob/main/eng/AndroidX.targets
-    This causes build error "Detected package downgrade":
-    https://learn.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu1605
-    Workaround: add a package reference to the higher version of the package. Possibly can be removed when MAUI bumps the implicit reference version.
-    -->
-    <PackageReference Include="Xamarin.AndroidX.Lifecycle.LiveData" Version="2.7.0.1" />
-
-    <!--Adding the above package reference requires Workaround for https://github.com/xamarin/AndroidX/issues/800#issuecomment-1774639627 -->
-    <!-- Tracking issue https://github.com/xamarin/AndroidX/issues/764 -->
-    <PackageReference Include="Xamarin.AndroidX.Collection" Version="1.4.0.1" />
-    <PackageReference Include="Xamarin.AndroidX.Collection.Ktx" Version="1.4.0.1" />
```

Comment:
Let's move this to a separate PR since it not related to `CommunityToolkit.Maui.MediaElement`

Reply:
I will try reverting the changes. It was only modified so that transient dependancies did not clash

Reply:
I just looked at the code and was unable to find the referenced code. It has been removed. Not sure when. Can you try updating your local source.